src/config.js

```diff
@@ -153,6 +157,8 @@ export async function discoverConfigFiles(
     path.join(getHomeDir(), `.${magicConfigName}`),
     // Look for a magic config in the current working directory.
     path.join(process.cwd(), magicConfigName),
+    // Look for webExt key inside package.json file
+    path.join(process.cwd(), 'package.json'),
```

Comment:
I think we should move this up one position in the array so that it happens before `path.join(process.cwd(), magicConfigName)`. This will make `package.json` load before `web-ext-config.js`. My rationale for this is that `package.json` is a project-wide type of configuration file and `web-ext-config.js` is a config file that is specific only to `web-ext`.

Reply:
That makes sense. I'll change that asap.